examples/advanced/geane/src/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+ ################################################################################
+ #    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+ #                                                                              #
+ #              This software is distributed under the terms of the             #
+ #              GNU Lesser General Public Licence (LGPL) version 3,             #
+ #                  copied verbatim in the file "LICENSE"                       #
+ ################################################################################
+# Create a library called "libPixel" which includes the source files given in
+# the array .
+# The extension is already found.  Any number of sources could be listed here.
+
+set(INCLUDE_DIRECTORIES
+  ${BASE_INCLUDE_DIRECTORIES}
+  ${CMAKE_SOURCE_DIR}/examples/advanced/geane/src
+  ${CMAKE_SOURCE_DIR}/examples/MQ/pixelDetector/src
+  ${CMAKE_SOURCE_DIR}/examples/common/mcstack
```

Comment:
The parts used from other examples should really be factored out and moved into an appropriate (either new or existing) FairRoot library. Otherwise, this will be a nightmare to maintain.